ejb3/src/main/java/org/jboss/as/ejb3/deployment/processors/EjbJndiBindingsDeploymentUnitProcessor.java

```diff
@@ -157,6 +159,11 @@ private void setupJNDIBindings(EJBComponentDescription sessionBean, DeploymentUn
                 }
                 logBinding(jndiBindingsLogMessage, remoteJNDIName);
             }
+
+            // log EJB's ejb:/ namespace binding
+            final String ejbNamespaceBindingName = ejbNamespaceBindingBaseName + "!" + viewClassName;
```

Comment:
If this is a stateful EJB it needs the '?stateful' string appended.

Reply:
Fixed for stateful EJB